forgerock-auth/src/main/java/org/forgerock/android/auth/devicebind/DeviceBindViewModel.kt

```diff
@@ -0,0 +1,70 @@
+package org.forgerock.android.auth.devicebind
+
+import android.content.Context
+import android.os.Parcelable
+import androidx.fragment.app.FragmentActivity
+import androidx.lifecycle.ViewModel
+import kotlinx.parcelize.Parcelize
+import org.forgerock.android.auth.InitProvider
+import org.forgerock.android.auth.callback.DeviceBindingAuthenticationType
+import org.json.JSONObject
+
+interface ViewModelHandler {
```

Comment:
Do we need an interface? I think this interface name is too generic.

Reply:
Are you taking about the purpose of this interface ?

Reply:
Yes, from the name ViewModelHandler, it sounds like handle all type of ViewModel, but this interface seems handling user's keys

Reply:
i see your point, let me think. The ViewModel class is designed to have some business logic, store and manage UI-related data . In our case when we have multiple user keys we will display the Fragment  an we set the current selected key in the viewmodel  .  May be UserKeyViewModel , UserKeyViewModelStore . Do you have any other thoughts around this ?